### DISCUSSION TITLE
Why is /dist not being ignored by git

### DISCUSSION THREAD

**Discussion by bricefriha** [axios/axios#6479] [Q&A] [answered]
I just set up the project, from my fork, to assist you with fixes and other things 🙂
I noticed something: the `/dist` folder is not included in the .gitignore, so changes are added every time I do an ``npm run build``.

Is it on purpose?
Just curious.

*sorry if it's not the right place to ask this question*

**Comment by jasonsaayman** [comment, marked as answer]
It is for operational reasons and it causes to many conflicts
